hknweb/settings/dev.py

```diff
@@ -6,19 +6,18 @@
 except ImportError:
     from .dummy_secrets import *
 
+import os
+
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['localhost','127.0.0.1','hkn.eecs.berkeley.edu','hkn.mu']
 
-INSTALLED_APPS += [
```

Comment:
I'm fine with removing livereload, but might be worthwhile to find a way to conditionally enable it with some flag.